core/src/test/java/org/apache/spark/sql/delta/UpdateJavaSuite.java

```diff
@@ -116,6 +116,27 @@ public void testWithConditionUsingColumn() {
         QueryTest$.MODULE$.checkAnswer(target.toDF(), expectedAnswer);
     }
 
+    @Test
```

Comment:
we don't need a duplicate test in Java as it doesn't increase the test coverage.